frontend/packages/client/src/components/WalletConnectModal.js

```diff
@@ -32,12 +32,8 @@ export default function WalletConnectModal({
       sortBy(
         services.map((service) => ({
           connectToService: () => {
-            if (service.uid === 'dapper-wallet#authn') {
-              // dapper wallet uses POP/RPC
-              fcl.config().put('discovery.wallet.method', 'POP/RPC');
-            } else {
-              // other wallets are fine with IFRAME/RPC
-              fcl.config().put('discovery.wallet.method', 'IFRAME/RPC');
+            if (service.uid !== 'Lilico') {
+              fcl.config().put('discovery.wallet.method', service.method);
```

Comment:
Blocto is deprecating IFRAME/RPC and switching to POP/RPC, so far it was been done on mainnet, this change will enable to use service method dynamically until mainnet and testnet are the same for blocto, then at some point all services will use 'POP/RPC'  excluding Lilico